src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import core.basesyntax.models.*;
+
+public class Main {
+    public static void main(String[] agrs) {

```

Comment:
There is a typo in the parameter name `agrs`. It should be `args` which is the conventional name for the main method arguments.